### PR TITLE
Added page for Mountain Madness 2022

### PR DESCRIPTION
**Pull request by SpartanApple**
Same validator errors are previous pages.

### CODE REVIEW

File: csss-site/src/events/mountain_madness/static/mountain_madness_static/2022/csss_custom.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.csss_fall_hacks_submission_row {
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/static/mountain_madness_static/2022/csss_custom.css:1]: is this actually used anywhere?

Comment (modernNeo) [inline csss-site/src/events/mountain_madness/static/mountain_madness_static/2022/csss_custom.css:1]: is this actually used anywhere?

**Review by modernNeo — CHANGES_REQUESTED** [review]
I think that `mountain_madness2022.html.swp` was incorrectly uploaded?

and are
`2022/logo-padded.svg`
`2022/sfu_logo.jpg`
`2022/sfu_logo.png`
actually used anywhere?

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2022/mountain_madness2022.html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+      </a>
+    </nav>
+
+    <section id='above-fold'>
+
+      <header>
+        <h3>The CSSS presents</h3>
+        <h1>Mountain Madness</h1>
+        <h3>April 29-30, 2022 @ Simon Fraser University</h3>
+        <h2><a href="https://www.eventbrite.ca/e/mountain-madness-hackathon-2022-tickets-321097068977?aff=Website" class='btn'>Register</a></h2>
```
Comment (modernNeo) [inline csss-site/src/events/mountain_madness/templates/mountain_madness/2022/mountain_madness2022.html:46]: is this link wrong?

**Review by modernNeo — CHANGES_REQUESTED** [review]

File: csss-site/src/events/mountain_madness/templates/mountain_madness/2022/mountain_madness2022.html (hunk 2 above)
Comment (SpartanApple) [reply csss-site/src/events/mountain_madness/templates/mountain_madness/2022/mountain_madness2022.html:46]: Link is right, site not published yet.

**Pull request merged**